docs/src/main/asciidoc/microprofile-graphql.adoc

```diff
@@ -631,6 +631,159 @@ query heroWithSurnames {
 }
 ----
 
+== Context
+
+You can get information about the GraphQL request anywhere in your code, using this experimental, SmallRye specific feature:
+
+[source,java]
+----
+@Inject
+Context context;
```

Comment:
I would have called the class `GraphqlContext` to be a bit more specific but I believe that ship has sailed :).

Reply:
Yea, none of the GraphQL Objects start with GraphQLXxxxx, so we kept it consistent.